src/index.test.tsx

```diff
@@ -100,7 +100,18 @@ test('Batching of subscriptions', async () => {
 
   render(<A rep={rep} />, div);
   await sleep(1);
-  expect(renderLog).to.deep.equal(['render A', null, 'render B', null, null]);
+  expect(renderLog).to.deep.equal([
```

Comment:
I have looked at this pretty hard and I am sure that this expectation is actually correct.

The subscription returns the default, then the query runs and re-fires the subscription, which re-renders.

I cannot explain why my change made it faster-enough to fit under the 1ms but somehow it does.

Reply:
I don't think this is right :-)

With the new code it renders twice. That is not needed. Nothing changed.

The subscribe body returns the same value as the default value so that should not trigger a render.